packages/ripple/src/index.ts

```diff
@@ -0,0 +1,119 @@
+import { getRippleGlobalNamespace } from './getRippleGlobalNamespace';
```

Comment:
We usually don't write the code within `index.ts` to make the file more findable. Can we move this code to a separate file or files like `registerRipple.ts`?

Reply:
👍